heron/spi/src/java/BUILD

```diff
@@ -146,3 +146,19 @@ java_library(
     ),
     deps = statemgr_deps_files,
 )
+
+java_binary(
+    name = "spi-unshaded",
+    srcs = glob([
+        "**/spi/utils/**/*.java",
+        "**/spi/common/**/*.java",
+    ]),
+    deps = utils_deps_files,
+)
+
+genrule(
+    name = "heron-spi-jar",
+    srcs = [":spi-unshaded_deploy.jar"],
+    outs = ["heron-spi.jar"],
+    cmd  = "cp $< $@",
+)
```

Comment:
We need to get this jar into heron-api install scripts - so that when a user installs he/she gets everything.